src/main/java/com/github/phuonghuynh/compressor/Sample.java

```diff
@@ -0,0 +1,10 @@
+package com.github.phuonghuynh.compressor;
+
+public final class Sample {
+  protected String name = "Fred";
+  
+  public void func() {
+     int anInt = 243;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "anInt" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "anInt". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)